src/app/shared/other/other.component.less

```diff
@@ -0,0 +1,19 @@
+@primary-red-color: #F40009;
```

Comment:
I believe we can create common less file with name "variables.less" and move all existed variables to this file.